src/applications/financial-status-report/wizard/pages/Decision.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import RadioButtons from '@department-of-veterans-affairs/component-library/RadioButtons';
+import { PAGE_NAMES } from '../constants';
+
+const label = 'Which of these issues do you want to report?';
+const options = [
+  {
+    value: 'disagree',
+    label: 'I disagree with the VA decision that resulted in this debt.',
+  },
+  {
+    value: 'error',
+    label: 'I this this debt is due to an error.',
+  },
+  {
+    value: 'wrong',
+    label: 'I think the amout of this debt is wrong.',
+  },
+];
+
+const Decision = ({ setPageState, state = {} }) => {
+  const setState = ({ value }) => {
+    switch (value) {
```

Comment:
Great use of a switch statement have been thinking a lot about switch vs ifs lately perfect use case